### PR TITLE
[doc feature][upgrade Teams with Outlook and Office]

### PR DESCRIPTION
**Pull request by v-vasudhab**
Please review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b6733355346889b9b049ab1aa2c309e3d629592e/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 83 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5f2be27e-1b2c-441b-9fd1-59caaee1dc0d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/07fa09616858979e4707cf99ab9c5225be458f3a/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 83 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/11ee05e7-4ede-402b-8507-6ebe54e5862b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0444ad16282571a36f64836d708d6cc3f976dc89/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 84 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/47b65c1e-99fb-452e-aa3b-3099bc878430) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eec6c5c48e6ddbc02936dde58b7ef7deb045e25c/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 86 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8b044706-89b6-43dc-94d1-c94d3a3273c6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ab54f9a9bd1f6b86e1166eff75400d4e8af68c7e/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 87 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cdff0ca4-7eca-48f9-adac-d8c8f3e8bfe0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ab54f9a9bd1f6b86e1166eff75400d4e8af68c7e/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 87 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2451a570-3468-49b5-9120-c03ade1b0fe0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5107e9e9228365603c85aa33aa50db6fbf26086c/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 85 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a394d9f0-9a01-47f9-9d9b-6a1035915c9c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/43286e9c1bdd6485775c06afa092b50f0922e410/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 85 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/20e973c2-af01-4de6-9964-2e2005389c6a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/674463515e5a2483abeafc4dd07a61656b69817b/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 85 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5e5aa864-d7e3-49a2-b6a9-4507e5a061c1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/150d00409194f663a35cefd7419b632fad945aa8/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 85 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e371ff85-ff85-486e-a5a6-b4191a3ee2a0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8092279d1df3d47ae986816592033c9016fc8839/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 85 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fe5a0ffe-e105-4691-a0a9-43ca5e96ce02) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f799d3173877754e40014c41c145fec957ecb353/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 85 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/236c4d31-5f4a-4486-992b-8edb82d2cca4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4ce8b0d31388c03856549a4f3733730f02175a54/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 85 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/21244ca9-cbaa-4d39-ad3b-b723a7e22bbc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+1. Select the command **Teams JS SDK to extend in Outlook and Office.com** to upgrade `TeamsJS` to the latest version.
+
+> [!NOTE]
+> Upgrading manifest file is required, but upgrading `TeamsJS` to latest version is optional, if you have already installed `TeamsJS` SDK in your application.
+
+> [!IMPORTANT]
+> If you have both `TeamsFx` and `TeamsJS` SDK packages in your Teams application,`TeamsFx` SDK is not currently compatible with the latest version (beta version) of `TeamsJS` SDK package. It can cause functional damage to your application when running in Outlook and Office.com.
+> If some functions are not working in Outlook or Office.com, remove `TeamsFx` packages from your application or don't upgrade `TeamsJS` SDK to latest version.
+
+## Prerequisite
```
Comment (v-abirade) [inline msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md:30]:
```suggestion
## Prerequisites
```

File: msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+Install the latest version of Teams Toolkit from Visual Studio Code extension in [Teams Toolkit (Preview) - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension).
+
+The following are the prerequisites to upgrade manifest and `TeamsJS` client SDK:
+
+### Upgrade manifest
+
+Run the command `upgrade Teams manifest` to work in Outlook and Office.com and select Teams Tab app manifest file to migrate:
```
Comment (v-abirade) [inline msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md:38]: Please provide an image to clarify or add steps for running the command.

File: msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+## Run Teams application in Outlook
+
+Perform the following steps to preview personal tab apps in Outlook web app and desktop clients:
+
+### Outlook web application
+
+1. Go to https://outlook.office.com 
+1. Select the three dots on the bottom left bar.
+
+    ![More apps view](../assets/images/upgrade-teams/apps.png)
```
Comment (v-abirade) [inline msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md:86]: The steps and the image do not match. Please update images using demo tenant ID and ensure the images are clear and the text content in the images are readable.

File: msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+1. Select the name of your app to preview in Outlook web application.
+
+    ![App preview in outlook](../assets/images/upgrade-teams/preview-outlook-web-application.png)
```
Comment (v-abirade) [inline msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md:90]: As informed in the previous comment, please change the image.

File: msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+### Outlook desktop client
+
+1. Open Outlook desktop client.
```
Comment (v-abirade) [inline msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md:94]: Please provide an image.

File: msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+1. Select the name of your app to preview it in Outlook Desktop Client.
+
+## Run Teams application in Office.com
+
+Perform the following steps to preview your apps in Outlook web client:
+
+1. Go to www.office.com/m365apps
+1. Find apps listed below “M365 Apps”
+
+    ![Apps in M365](../assets/images/upgrade-teams/m365-app.png)
```
Comment (v-abirade) [inline msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md:105]: As discussed and mentioned in the comment above, please change the two images in this section.

**Review by v-abirade — CHANGES_REQUESTED** [review]
@surbhigupta12, I have updated review comments, could you please look into it.

File: msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+# Upgrade Teams application to work with Outlook and Office.com
+
+> [!NOTE]
+> This feature is currently available in Developer preview only.
+
+Teams Toolkit helps you to upgrade Teams applications to work with Outlook and Office.com. You must upgrade the manifest and `TeamsJS` SDK to the latest version to run the application on Outlook and Office.com, to extend your Teams applications in Outlook and Office.com, the migration commands in Teams are as follows:
+
+![Migration of Teams](../assets/images/upgrade-teams/teams-extended-in-outlook-and-office.png)
+
+1. Select the command **Teams manifest to extend in Outlook and Office.com** to upgrade manifest and communicate with Office and Outlook:
```
[the remark this answers is not in the document]
Comment (MuyangAmigo) [reply msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md:19]: Can you link this part to Erika's PR?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/07ca02469837178082468185b7835177124ce04d/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 82 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cf4db052-6ef6-44bb-a5f6-776e35922ec1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f36cfff4437a6143bcc55fa02ec775b2adf71b02/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 82 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ef44e3a2-2fd3-4ca7-967c-5768a43e0db9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/06ba3a44ff65b30e3a30087a1c9579dbce8228f2/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 86 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/11548603-54ff-4704-9b20-ca4d5719daae) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1842675034ddd9eea79247a4650dd2728c5b3d04/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 86 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6114149b-77d9-4367-998d-294b0b5092af) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (v-abirade) [reply msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md]: We can only add links once her document is merged. Once Surbhi merger Erika's PR, I will add the link.

**Comment by surbhigupta12** [comment]
@v-abirade,

Erika's PR is green. Please resolve all warning/suggestions for this PR and finalize it by first half.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/af8f2768d3c0b7f81b30fed2ea3f70b68ce35889/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 86 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7d80abca-24c7-4398-b529-4b3f5304b531) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/78db0baa70b985f03d63141d332294d36ac1c818/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 86 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/70de9fea-5158-4d51-9cbc-1d7763243258) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bc6d68440f1e8759809f882d930c826e4f829f8b/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 85 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/140cf870-538f-4015-94f1-c052d1865dad) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1255fdc7e0fadfb27f92b42b89efbbd1620ffa91/msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and%20office.md) | 85 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/625375e5-8da4-4724-9577-b88fd5484bb3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by surbhigupta12** [comment]
As informed by Yu (@zyxiaoyuer), closing this PR as the requirement no longer exists.

**Pull request closed without merging**